modules/RTC/JitsiTrack.js

```diff
@@ -79,13 +80,38 @@ function JitsiTrack(conference, stream, track, streamInactiveHandler, trackMedia
      * @type {boolean}
      */
     this.disposed = false;
+    this._setHandler("inactive", streamInactiveHandler);
+}
 
-    if(stream) {
-        if (RTCBrowserType.isFirefox()) {
-            implementOnEndedHandling(this);
+/**
+ * Sets handler to the WebRTC MediaStream or MediaStreamTrack object depending
+ * on the passed type.
+ * @param type {string} the type of the handler that is going to be set
+ * @param handler {Function} the handler.
+ */
+JitsiTrack.prototype._setHandler = function (type, handler) {
+    if(this.stream) {
+        if(type === "inactive") {
+            if (RTCBrowserType.isFirefox()) {
+                implementOnEndedHandling(this);
+            }
+            addMediaStreamInactiveHandler(this.stream, handler);
         }
-        addMediaStreamInactiveHandler(stream, streamInactiveHandler);
     }
+    this.handlers[type] = handler;
+}
+
+/**
+ * Sets the stream property of JitsiTrack object and sets all stored handlers
+ * to it.
+ * @param stream {MediaStream} the new stream.
```

Comment:
The correct docs syntax will be:
``
@param {MediaStream} stream the new stream
``